test_oauth.sh

```
@@ -0,0 +1 @@
+cargo run -- auth login > oauth_output.txt 2>&1 &
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)